rules/no-ignored-test-files.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+var path = require('path');
+var pkgUp = require('pkg-up');
+var multimatch = require('multimatch');
+var util = require('../util');
+var createAvaRule = require('../create-ava-rule');
+
+var defaultFiles = [
+	'test.js',
+	'test-*.js',
+	'test/**/*.js'
+];
+
+function isInIgnoredFolder(relativeFilePath) {
+	var folders = relativeFilePath.split(path.sep);
+	var ignoredFolder;
+	if (folders.indexOf('fixtures') !== -1) {
+		ignoredFolder = 'fixtures';
+	} else if (folders.indexOf('helpers') !== -1) {
+		ignoredFolder = 'helpers';
+	}
+
+	if (ignoredFolder) {
```

Comment:
Sounds good! Will do that (won't put node_modules though as ESLint ignores them already)